### ISSUE TITLE
Time domain plots are too sparse

### ISSUE THREAD

**Issue by hagenw** [sfstoolbox/sfs-python#170]
The example images in [master](https://sfs-python.readthedocs.io/en/latest/sfs.td.wfs.html) are broken compared compared to [v0.5.0](https://sfs-python.readthedocs.io/en/0.5.0/sfs.td.wfs.html).

E.g.

![image](https://user-images.githubusercontent.com/173624/105148085-458f1800-5b02-11eb-9de8-a3a78e549b15.png)

vs.

![image](https://user-images.githubusercontent.com/173624/105148055-37d99280-5b02-11eb-904e-e17ce29b7e30.png)

**Comment by fs446** [comment]
I spent the day to figure out what happens here. I could not 100% trace it, but it seems that matplotlib rendering changed in the meantime between these two versions. One could figure out this exactly, however our SFS processing is not optimum anyway, so we should rather address this first. The problem is that by using a perfect Dirac impuls as excitation the synthesized sound field has very sharp transients and a lot of zeros (in dB we then get -Inf) and the rendering algorithms need somehow align these values to RGB. Success is then highly dependent on the sampling grid, amplitude, time instance and so on, which can be observed, when playing around with different configurations.

So, to solve the problem in the toolbox I would propose to use bandlimited Dirac impulse as excitation, see the branch
https://github.com/sfstoolbox/sfs-python/tree/doc_td_plot_routine
where a simple Kaiser-Bessel windowed FIR lowpass is used with some meaningful parameters for nice plots.

We might consider to write a signal generating function, e.g. create_bandlimited_dirac(number of samples, cut frequency, **kwargs) for user convenience.

Furthermore, I would propose to use the normalization of weights for the points sources, such that we get similar level than for the plane wave case. Normalization for focused source seems to be not correct yet.

**Comment by hagenw** [comment]
Good point, I think we did something similar in Matlab and used a hanning window as Dirca impulse at some time, see https://github.com/sfstoolbox/sfs-matlab/blob/02194f0243d1ead26572f760032c40527718919d/SFS_time_domain/dirac_imp.m#L48-L53

**Comment by hagenw** [comment]
> So, to solve the problem in the toolbox I would propose to use bandlimited Dirac impulse as excitation, see the branch
> https://github.com/sfstoolbox/sfs-python/tree/doc_td_plot_routine
> where a simple Kaiser-Bessel windowed FIR lowpass is used with some meaningful parameters for nice plots.

Seems to work, for me locally the WFS plane wave plot from above now looks like this:

![image](https://user-images.githubusercontent.com/173624/105318317-a6caf000-5bc3-11eb-9d80-3f572c048342.png)

**Comment by fs446** [comment]
in commit a175ad8196ef5967df788750afc2919850119218 I added a low-passed filtered Dirac such that the wavefronts are smeared a little bit.
